easybuild/easyconfigs/p/Python/Python-3.8.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,431 @@
+name = 'Python'
+version = '3.8.4'
```

Comment:
@Micket 3.8.6 was released Sept 24th, any reason to stick to this older 3.8 version?!

Reply:
I must have misread the webpage or something. Bumped to 3.8.6